react/features/app/components/App.native.js

```diff
@@ -109,7 +109,13 @@ export class App extends AbstractApp {
                 }
             }
 
-            dispatch(updateSettings(this.props.userInfo || {}));
+            const userInfo = { ...this.props.userInfo } || {};
+
+            if (!userInfo.avatarURL) {
```

Comment:
please add a comment like 'making sure avatarURL is not null, but either valid or undefined